KS Auto Service/css/header.css

```diff
@@ -0,0 +1,80 @@
+.main-block {
+  background: url(../img/img-general.jpg);
+  background-color: var(--center-background);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-top: 125px;
+}
+
+.head {
+  position: fixed;
+  top: 0;
+  background-color: var(--center-background);
+  z-index: 1;
```

Comment:
should be bigger number
to be sure the header will be above any other element

`z-index: 1` don't give you such confidence at all, because any element even with the same z-index will be above of header because they are written lower in html than header